### PR TITLE
fix(listbox): inline size

### PR DESCRIPTION
**Pull request by rinaok**

### CODE REVIEW

File: libs/components/src/lib/listbox/listbox.scss [hunk 1]
```diff
@@ -15,6 +15,7 @@
 
 :host {
 	display: block;
+	inline-size: fit-content;
```
Comment (YonatanKra) [inline libs/components/src/lib/listbox/listbox.scss:18]: Doesn't fit content come with `max size`?
E.g. `fit-content(--vvd-listbox-max-size)`

Comment (rachelbt) [reply libs/components/src/lib/listbox/listbox.scss:18]: @YonatanKra when its on the host the consumer can change it :)

**Review by yinonov — DISMISSED** [review]
reduce ui test image size

**Comment by rinaok** [comment]
> reduce ui test image size

@yinonov it's ok now?

**Review by rachelbt — APPROVED** [review]
LGTM

**Pull request merged**